settings.php

```diff
@@ -187,32 +190,40 @@
 ini_set('session.gc_divisor', 100);
 
 /**
- * Set session lifetime (in seconds), i.e. the time from the user's last visit
- * to the active session may be deleted by the session garbage collector. When
- * a session is deleted, authenticated users are logged out, and the contents
- * of the user's $_SESSION variable is discarded.
+ * Set session lifetime (in seconds).
+ *
+ * I.e. the time from the user's last visit to the active session may be deleted
```

Comment:
I would just use *It is the last time from the user's last visit to the active session may be deleted by the session garbage collector.* There is no need to use *i.e.* as it is clear that part is explaining what *session lifetime* is.

(As a side note, as a non-native English speaker, I find *It is the last time from the user's last visit to the active session may be deleted by the session garbage collector.* difficult to parse, probably because I am not clear what may be deleted. I cannot understand where the subordinate clause starts and where the main clause ends.)